docs/content/deployment/next-steps/_index.md

```diff
@@ -26,39 +26,46 @@ Active Choria configuration:
 
 The active configuration used in Choria comes from using Puppet AIO defaults, querying SRV
 records and reading configuration files.  The below information shows the completely resolved
-configuration that will be used when running Choria commands
+configuration that will be used when running MCollective commands
```

Comment:
This come from:
https://github.com/smortex/mcollective-choria/blob/discover-using-choria-service/lib/mcollective/application/choria.rb#L123